models/fivetran_log__connector_status.sql

```diff
@@ -95,7 +110,8 @@ connector_health as (
             when last_sync_completed_at is null and last_error_at is null then 'initial sync in progress'
 
             -- there's been an error since the connector last completed a sync
-            when last_error_at > last_sync_completed_at then 'broken'
+            when --last_error_at > last_sync_completed_at and 
+                last_sync_batch_id = last_error_batch then 'broken'
```

Comment:
reminder to remove commented out code

i'd also maybe change the comment in line 112 to like `the last attempted sync had an error`